core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -575,7 +592,7 @@ namespace kagome::consensus::babe {
     if (auto next_epoch_digest_res = getNextEpochDigest(block.header);
         next_epoch_digest_res) {
       auto &next_epoch_digest = next_epoch_digest_res.value();
-      SL_INFO(log_,
+      SL_VERBOSE(log_,
```

Comment:
I suppose it could be more coherent to replace other log_->verbose() calls here with this macro, too, but it's not something really significant.